README.md

```diff
@@ -5,63 +5,110 @@
 
 <h1 align="center">MUI X</h1>
 
-The [main repository](https://github.com/mui/material-ui) of MUI focuses on providing a ready-to-use design system, as well as empowering developers to build their own.
+MUI X is a collection of advanced UI components, including a [data grid](https://mui.com/x/react-data-grid/) and [date and time pickers](https://mui.com/x/react-date-pickers/getting-started/).
 
-This MUI X mono-repo gathers the advanced MUI components – it's an extension of the core.
-The fundamental features are MIT licensed, and the more advanced ones commercial.
+These components are significantly more complex than the ones found in the [MUI Core](https://github.com/mui/material-ui/) libraries.
+They feature advanced functionality for data-rich applications and a wide range of other use cases.
+
+MUI X is **open core**—base components are MIT-licensed, while more advanced features require a Pro or Premium commercial license.
+See the [Licensing page](https://mui.com/x/introduction/licensing/) for details.
 
 ## Components
 
-- [Data grid](https://mui.com/x/react-data-grid/)
-- [Date / Time pickers](https://mui.com/x/react-date-pickers/getting-started/)
+- [Data Grid](https://mui.com/x/react-data-grid/)
+- [Date and Time Pickers](https://mui.com/x/react-date-pickers/getting-started/)
+
+## Installation
+
+### Data Grid
+
+Read the Data Grid [Installation instructions](https://mui.com/x/react-data-grid/getting-started/#installation) in the MUI X docs.
+
+### Date and Time Pickers
```

Comment:
```suggestion
### Date and Time pickers
```

Reply:
This kind of capitalization looks very awkward to me. Is it title case? Is it sentence case? It seems to be somewhere in between. I have proposed here https://github.com/mui/mui-x/issues/5764#issuecomment-1213536072 that we refer to Date and Time Pickers as a proper noun, always capitalized, unless referring to the general concept of these components. Same for Data Grid—it seems weird to me that we wouldn't treat our product names as Proper Nouns in title case. It would be like writing "MUI base."

Reply:
I'd also vote for going with `title case` for component names, but that would mean replacing all of them in both `mui` and `mui-x` repositories and in such case we need to decide if it's even worth it..? 🤔

Reply:
I think it's worth it for the sake of consistency and professionalism. It's something I've begun to address as I revise individual pages in #5764—see #5780 and #5781—but I haven't moved forward with it since we still don't have a consensus on this point.

Reply:
@samuelsycamore I didn't know we started a discussion on changing this convention. I have been making changes for a couple of months in the opposite direction, e.g. in Material UI https://github.com/mui/material-ui/pull/32238 😁. I think that my rationale was:

1. sentence case is simpler
2. used by MD https://material.io/components/data-tables
3. It makes it easier to understand if you mean the component data grid or the React component DataGrid (Data Grid being visually closer, hence more confusing)

Now, I do agree, when you look at the Google search results, you see the Title case. Google is changing the capitalization (caught by ahrefs) of the pages listed.

No objections from my end, as long as we update everything. cc @mbrookes maybe for thoughts as well.

Reply:
Let's do it!

Reply:
OK, so we make a new exception to our rule. We always use sentence case as the default, but we use Title Case for roles, e.g. Developer Advocate, and for component names, e.g. Text Field.